resource-rating-plugin-functionality/resource-rating-plugin-metaboxes.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+* Plugin Name: Resource Rating
```

Comment:
Please remove the comments, as this should only be included in the main file which is `resource-rating-plugin.php`